apps/dav/lib/CalDAV/Reminder/NotificationProvider/EmailProvider.php

```diff
@@ -80,15 +81,24 @@ public function __construct(IConfig $config,
 	 * @param VEvent $vevent
 	 * @param string $calendarDisplayName
 	 * @param array $users
+	 * @param IUser $userOfReminder
 	 * @throws \Exception
 	 */
 	public function send(VEvent $vevent,
 						 string $calendarDisplayName,
-						 array $users = []):void {
+						 array $users = [],
+						 IUser $userOfReminder = null):void {
 		$fallbackLanguage = $this->getFallbackLanguage();
 
 		$emailAddressesOfSharees = $this->getEMailAddressesOfAllUsersWithWriteAccessToCalendar($users);
-		$emailAddressesOfAttendees = $this->getAllEMailAddressesFromEvent($vevent);
+
+		$organizer = $this->getOrganizerEMailAndNameFromEvent($vevent);
+
+		$emailAddressesOfAttendees = [];
+
+		if ($userOfReminder && strcasecmp($userOfReminder->getEMailAddress(), key($organizer)) == 0) {
```

Comment:
Make sure both `$userOfReminder` and `$organizer` are not `null`.
```suggestion
		if ($organizer !== null && $userOfReminder instanceof IUser && strcasecmp($userOfReminder->getEMailAddress(), key($organizer)) === 0) {
```

Reply:
:+1: